src/InteractivePathAnalysis/CustomMenu.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <QWidget>
+
+#include <QMouseEvent>
+
+class QPushButton;
+class QVBoxLayout;
+
+/**
+ * @brief Simple Menu Implementation
+ * 
+ * Behaves like a regular menu but allows any type of widgets in the menu body.
+ */
+class CustomMenu final : public QWidget
+{
+    Q_OBJECT
+public:
+    enum Alignment {

```

Comment:
Is it possible to use Qt::Alignment here?

Reply:
> Is it possible to replace with Qt::Alignment ?

corrected